frontend/src/app/insight/page.tsx

```diff
@@ -0,0 +1,116 @@
+import "globals";
```

Comment:
Remove this please, `globals.css` is already imported in `app/layout.tsx`

Reply:
Done! Won't repeat again!!